src/app/app.routes.ts

```diff
@@ -1,15 +1,20 @@
 import { Routes } from '@angular/router';
 import { HomeComponent } from './pages/home/home.component';
+import { MenuComponent } from './components/menu/menu/menu.component';
+import { KittensComponent } from './pages/kittens/kittens/kittens.component';
+import { SigninComponent } from './components/signin/signin/signin.component';
+import { SignupComponent } from './components/signup/signup/signup.component';
+import { ContactComponent } from './pages/contact/contact/contact.component';
+import { AboutComponent } from './pages/about/about/about.component';
 
 export const routes: Routes = [
-  {
-    path: 'home',
-    component: HomeComponent,
-  },
-  {
-    path: '',
-    redirectTo: '/home',
-    pathMatch: 'full',
-  },
-
+  {    path: 'home', component: HomeComponent, },
```

Comment:
french name for urls content, english for code

Reply:
done!